cmd/wallet-web/src/components/Vaults/TheRenameVaultModal.vue

```diff
@@ -60,13 +60,13 @@ import { computed, ref, watch } from 'vue';
 import { useStore } from 'vuex';
 import { CollectionManager } from '@trustbloc/wallet-sdk';
 import { useI18n } from 'vue-i18n';
-import { vaultsMutations } from '@/pages/Vaults.vue';
-import Modal from '@/components/Modal/Modal.vue';
-import InputField from '@/components/InputField/InputField';
-import StyledButton from '@/components/StyledButton/StyledButton';
+import { vaultsMutations } from '@/pages/VaultsPage.vue';
+import Modal from '@/components/Modal/TheModal.vue';
+import InputField from '@/components/InputField/TheInputField';
+import StyledButton from '@/components/StyledButton/TheStyledButton';
```

Comment:
You'd need to update the import objects as well:
```suggestion
import TheModal from '@/components/Modal/TheModal.vue';
import TheInputField from '@/components/InputField/TheInputField';
import TheStyledButton from '@/components/StyledButton/TheStyledButton';
```